.github/workflows/pr_build.yaml

```diff
@@ -32,17 +32,9 @@ jobs:
       
     steps:
     - uses: actions/checkout@v2
-      
-    - name: Setup dotnet
-      uses: actions/setup-dotnet@v1.9.0
-      with:
-        dotnet-version: 6.0.x
-
-    - name: Setup MSBuild
-      uses: microsoft/setup-msbuild@v1.1
 
-    - name: Setup NuGet
-      uses: NuGet/setup-nuget@v1.0.5
+    - name: Setup build dependencies
+      uses: ./.github/workflows/composite/bootstrap
```

Comment:
Could these composite components be hosted on a separate Gist for repo, so users of this generic script didn't have to include them in their own .github folder?

Reply:
Yes, you can have a separate repo for these.  Not many people care about these bootstraps being merged like this tbh, but yeah you could make a separate action that does this.

Reply:
Hmm, the repo composite not working...need to debug more and doing that on the side...